src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -604,6 +603,11 @@ export default class PeopleAndRoles extends Mixins(CommonMixin, DateMixin, OrgPe
     }
   }
 
+  /** Returns True if the orgPerson's Business Number has changed. */
+  private hasBnChanged (orgPerson: OrgPersonIF): boolean {
+    return (orgPerson.officer.taxId !== this.originalParties[this.activeIndex].officer.taxId)
```

Comment:
This is for a new "BN CHANGED" v-chip.

This is to be consistent with showing all changes to a pre-existing org-person.